ruby/alias_manager.rb

```diff
@@ -0,0 +1,24 @@
+#Swap the first and last name
+
+#Change all vowels (a,e,i,o,u) to the next vowel in 'aeiou'. Change all 
+#consonants to the next consonant.
+
+attempts = 0
+
+aliases = []
+
+until attempts == 3
+	attempts += 1
+
+puts "Please list your full name (first and last name)."
+full_name = gets.chomp.downcase!
+
+full_name.tr!('aeiou', 'eioua') 
```

Comment:
using `.tr!` is clever but think of this: you want this method to be reusable and not have to use the vowel and consonant strings over and over again by typing them out. What is a way you could make these vowel and consonant strings reusable and callable? Variables! You could also encapsulate all of this in a method, like:
```
def change_name(fullname)
    vowels = 'aeiou'
    shifted_vowels = 'eioua'
    consonants = 'bcdfghjklmnpqrstvwxyz'
    shifted_consonants = 'cfghjklmnpqrstvwxyzb'
    fullname.tr!(vowels, shifted_vowels)
    fullname.tr!(consonants, shifted_consonants)
    fullname
end
```